src/ImportDefinitionsBundle/Interpreter/HrefInterpreter.php

```diff
@@ -30,14 +31,20 @@ class HrefInterpreter implements InterpreterInterface, DataSetAwareInterface
      */
     public function interpret(Concrete $object, $value, Mapping $map, $data, DefinitionInterface $definition, $params, $configuration)
     {
+        $type = $configuration['type'];
+
+        if ($type && $value) {
+            return Service::getElementById($type, $value);
+        }
+
         $objectClass = $configuration['class'];
```

Comment:
and only do this if the type is object?

Reply:
If the type is object, the Service::getElementById handle it already.
So if the class is set, the Service:getElementById shouldn't handle it?

Reply:
no, since it bypasses the class configuration